packages/bpk-component-breakpoint/src/BpkBreakpoint.tsx

```diff
@@ -45,6 +45,18 @@ type Props = {
   matchSSR?: boolean;
 };
 
+const useLegacyWarning = (query: string, legacy: boolean, isClient: boolean) =>
+  useMemo(() => {
```

Comment:
Ensures that the `console.log` only occurs once per query/instance to not duplicate the console output (as we are forcing a re-render)